### PR TITLE
[WIP] Setup.py support (attempt 2)

### PR DESCRIPTION
**Pull request by teogramm**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    with tarfile.open("package.tar", "w") as package:
+        package.add("scripts")
+        package.add("bash_helpers")
+
+
+package_artifacts()
+os.rename("package.tar", "genume/package.tar")
+setuptools.setup(
+    name="genume",
+    version="0.1.1",
```
Comment (Sima214) [inline setup.py]: Actually 0.1.2

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            'genume = genume.console:console_entry'
+        ]
+    },
+    classfilters=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
+        "Operating System :: POSIX",
+    ],
+)
+os.remove("genume/package.tar")
```
Comment (Sima214) [inline setup.py]: New line required(pycodestyle)

File: genume/console.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from genume.logging.logger import init as initLogger
```
Comment (Sima214) [inline genume/console.py]: Duplicate of __main__.
Just move console_entry into __main__.
No need for duplicating startup code.

**Comment by xlxs4** [comment]
:+1:

**Comment by teogramm** [comment]
Fixed!

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    name="genume",
+    version="0.1.2",
+    author="CSD FOSS Team",
+    description="A graphical enumeration tool",
+    url="https://github.com/CSD-FOSS-Team/genume",
+    packages=setuptools.find_packages(),
+    include_package_data=True,
+    entry_points={
+        'console_scripts': [
+            'genume = genume.genume:main'
```
Comment (Sima214) [inline setup.py]: Wrong main.

**Comment by xlxs4** [comment]
@teogramm How's it going?

**Comment by teogramm** [comment]
Created a new function to be used as entrypoint

**Comment by Sima214** [comment]
I will check it tomorrow.
PS: the current way to update your fork is by running `make update`.

**Comment by teogramm** [comment]
Thanks. I will keep that in my mind as my fork is somewhat of a mess currently. The helper function was the most elegant solution I could think of.

**Comment by Sima214** [comment]
Merged, but github did not recognize it because I rebased your commits.

**Pull request closed without merging**